docs/book/src/user/quick-start.md

```diff
@@ -463,6 +463,18 @@ export SERVICE_DOMAIN="k8s.test"
 {{#/tab }}
 {{#tab GCP}}
 
+
+```bash
+# Name of the GCP datacenter location. Change this value to your desired location
+export GCP_REGION=us-east1-b
+# Name of the GCP project. Should be the same as that used in gcp-credentials.json
+export GCP_PROJECT=sonic-base-322703
+
+# Select instance types
+export GCP_CONTROL_PLANE_MACHINE_TYPE:=e2-medium
```

Comment:
```suggestion
export GCP_CONTROL_PLANE_MACHINE_TYPE=e2-medium
```

Reply:
@cpanato can help review it. I don't know what's the prow command to add him as reviewer.

Reply:
/cc @cpanato